onedal/basic_statistics/basic_statistics.py

```diff
@@ -75,8 +77,12 @@ def _compute_raw(self, data_table, weights_table, module, policy, dtype=np.float
     def _compute(self, data, weights, module, queue):
         policy = self._get_policy(queue, data, weights)
 
-        if not (data is None):
+        type_name = type(data).__name__
+        is_sparse = False
+        if not (data is None) and type_name != "csr_matrix" and type_name != "csr_array":
```

Comment:
Is this common for checking sparsity?

Reply:
If yes please create common primitive and use it in code base.

Reply:
Maybe I have to rename `is_sparse` to `is_csr` as we need to check here that the data is in CSR format exactly (not CSC, COO or any other sparse formats). And there are only 2 data types in SciPy for now that represent CSR data.

[scipy.sparse.csr_array.getformat()](https://docs.scipy.org/doc/scipy/reference/generated/scipy.sparse.csr_array.getformat.html#scipy.sparse.csr_array.getformat) API cannot be used as it is deprecated for csr_array and will be soon removed. And there is no alternative that would work for both csr_matrix and csr_array.